.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 15
+    timeout-minutes: 20
```

Comment:
The ubuntu 3.10 test runner that downloads data and runs some extra tests was _just_ timing out at 15 minutes (it succeeds in 15 min, 0 seconds 🤦)
![Screenshot 2025-05-20 at 11 19 23 AM](https://github.com/user-attachments/assets/ff7d6f81-6562-40d9-b933-ea63d9c44ac1)

Reply:
I think we should remove the timeout-minutes, not sure why it was added as I didn't follow that addition. We are on the free tier and not paying for these minutes.